src/lib/DatePicker.svelte

```diff
@@ -7,19 +7,29 @@
 
   const dispatch = createEventDispatcher<{ select: undefined }>()
 
+  /** Default Date to use */
+  const defaultDate = new Date()
+
   /** Date value */
-  export let value = new Date()
+  export let value: Date | null = null;
   function setValue(d: Date) {
-    if (d.getTime() !== value.getTime()) value = d
+    if (d.getTime() !== value?.getTime()) value = d
+  }
+
+  tmpPickerDate = value ?? defaultDate;
+  $: tmpPickerDate = value ?? defaultDate;
```

Comment:
Instead of having the `tmpPickerDate` variable, maybe we could have a function like this:
```ts
function toPickerDate(value: Date | null) {
  if (value === null) {
    return defaultDate
  } else {
    return value
  }
}
```

Reply:
> are you thinking we replace the variable with the function call everywhere it gets referenced?

Yes, that was the idea, so you wouldn't have 2 different variables for the value

Reply:
nvm, you need to be able to assign to `tmpPickerDate` to change the date picker month/year without updating the `value`